radish34/messenger/config/development.js

```diff
@@ -5,9 +5,6 @@ module.exports = {
     bufferMaxEntries: 8,
     firstConnectRetryDelaySecs: 5,
   },
-  logging: {
```

Comment:
So where do we set the log level if not here?

Reply:
Am I right saying this configs only hold for the messenger services? So what about the other services? A common way to handle this would be nice, so any suggestions?

Reply:
Agree this needs to live in a common package. We have an integration branch that stands up these shared artifacts called `init-core` @myronrotter. Let's get this merged into master and then we can figure out how to best expose it in shared utils package of some kind.